AElf.Cryptography/OpenAccount.cs

```diff
@@ -6,7 +6,8 @@ namespace AElf.Cryptography
 {
     public class OpenAccount
     {
-        public Timer Timer { get; set; }
```

Comment:
The field `Timer` does not give any information to code readers. Change it to CloseTimer and add some comments.